src/spacing.ts

```diff
@@ -0,0 +1,37 @@
+import {Property as CSS} from 'csstype'
+import {system, Config, ResponsiveValue, ThemeType} from '@styled-system/core'
+
+export interface SpacingProps<Theme extends ThemeType = ThemeType>{
+    m?: ResponsiveValue<CSS.Background, Theme>
+    mt?: ResponsiveValue<CSS.Background, Theme>
+    mr?: ResponsiveValue<CSS.Background, Theme>
+    mb?: ResponsiveValue<CSS.Background, Theme>
+    ml?: ResponsiveValue<CSS.Background, Theme>
+    mx?: ResponsiveValue<CSS.Background, Theme>
+    my?: ResponsiveValue<CSS.Background, Theme>
+    p?: ResponsiveValue<CSS.Background, Theme>
+    pt?: ResponsiveValue<CSS.Background, Theme>
+    pr?: ResponsiveValue<CSS.Background, Theme>
+    pb?: ResponsiveValue<CSS.Background, Theme>
+    pl?: ResponsiveValue<CSS.Background, Theme>
+    px?: ResponsiveValue<CSS.Background, Theme>
+    py?: ResponsiveValue<CSS.Background, Theme>
```

Comment:
```suggestion
    m?: ResponsiveValue<CSS.Background, Theme>
    mt?: ResponsiveValue<CSS.Background, Theme>
    mr?: ResponsiveValue<CSS.Background, Theme>
    mb?: ResponsiveValue<CSS.Background, Theme>
    ml?: ResponsiveValue<CSS.Background, Theme>
    mx?: ResponsiveValue<CSS.Background, Theme>
    my?: ResponsiveValue<CSS.Background, Theme>
    p?: ResponsiveValue<CSS.Background, Theme>
    pt?: ResponsiveValue<CSS.Background, Theme>
    pr?: ResponsiveValue<CSS.Background, Theme>
    pb?: ResponsiveValue<CSS.Background, Theme>
    pl?: ResponsiveValue<CSS.Background, Theme>
    px?: ResponsiveValue<CSS.Background, Theme>
    py?: ResponsiveValue<CSS.Background, Theme>
  ```

ini masih salah mas @heriprastiio 
harusnya `py?: ResponsiveValue<CSS.Padding, Theme>`

pakai Padding atau Margin untuk CSS.<Type> nya

Reply:
Saya mengerjakan spacing, mohon direview kembali